### PR TITLE
[CHORE] Remove redundant GameState fields.

### PR DESCRIPTION
**Pull request by AzaroQL**
# Description

Some minor cleanup.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manual test + yarn test

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VITE_API_URL=<API_URL>
+# VITE_API_URL=https://api-dev.sunflower-land.com
```
Comment (adamhannigan) [inline .env.sample:1]: We don't want to expose this URL in our repo - it makes it an easy target for bad actors 😓

File: src/features/bumpkins/components/DynamicNFT.tsx [hunk 2]
```diff
@@ -24,15 +24,22 @@ export const DynamicNFT: React.FC<Props> = ({
...
 
   useEffect(() => {
+    let isSubscribed = true;
     const load = async () => {
       const image = await buildImage({
         parts,
       });
 
-      setImageSrc(image);
+      if (isSubscribed) {
```
Comment (adamhannigan) [inline src/features/bumpkins/components/DynamicNFT.tsx:33]: What is this used for?

**Review by adamhannigan — COMMENTED** [review]
LGTM 🚀 @AzaroQL 

Just a small question and suggestion to remove the dev API url 😅

File: .env.sample (hunk 1 above)
Comment (AzaroQL) [reply .env.sample]: Done

File: src/features/bumpkins/components/DynamicNFT.tsx (hunk 2 above)
Comment (AzaroQL) [reply src/features/bumpkins/components/DynamicNFT.tsx:33]: To prevent setting state on an unmounted component if async task finishes late.

**Review by adamhannigan — APPROVED** [review]
LGTM

**Pull request merged**